### PR TITLE
Workaround for iOS 13 navigation bar height issue

### PR DESCRIPTION
**Pull request by enhorn**
Workaround for this iOS 13 issue: https://github.com/iZettle/Presentation/issues/49

Before:
NavBar

After:
Fixed

### CODE REVIEW

File: Presentation/UINavigationController+Presenting.swift [hunk 1]
```diff
@@ -162,7 +162,14 @@ private extension UINavigationController {
 
 
 
-        if let coordinator = transitionCoordinator, !animated {
+        if #available(iOS 13.0, *), viewControllers.isEmpty {
+            // iOS 13 has an issue where the NavBar gets too narrow at initial presentation.
+            // Have tried different approaches to force UIKit to update the navbar layout on the current layout pass, but found no working solution.
```
Comment (TBernatskaya) [inline Presentation/UINavigationController+Presenting.swift]: I'm wondering if it's connected to ```scrollEdgeAppearance```.

Comment (TBernatskaya) [reply Presentation/UINavigationController+Presenting.swift]: for ex. :
```
    if #available(iOS 13.0, *), viewControllers.isEmpty {
        navigationBar?.scrollEdgeAppearance = .standardAppearance
    } else {
        ...
    }```

Comment (enhorn) [reply Presentation/UINavigationController+Presenting.swift]: I'll try it out!

Comment (enhorn) [reply Presentation/UINavigationController+Presenting.swift]: @TBernatskaya  Didn't do any difference. :-/

**Comment by enhorn** [comment]
Waiting to we if this bug has been in the GM release of iOS 13, that probably will be released in a few hours.

**Comment by enhorn** [comment]
The issue is still there in the GM. :-/

**Pull request merged**